assets/src/edit-story/components/canvas/page.js

```diff
@@ -53,37 +54,42 @@ function Page() {
 		}
 		evt.stopPropagation();
 
+		// Persist the event so that a dragstart could be
+		// later triggered without having to select the element first.
 		if ( 'pointerdown' === evt.type ) {
 			evt.persist();
 			setPushEvent( evt );
 		}
 	}, [ editingElement, clearEditing, toggleElementIdInSelection, selectElementById ] );
 
-	const selectedElement = selectedElements.length === 1 ? selectedElements[ 0 ] : null;
+	const singleSelection = 1 === selectedElements.length;
+	const hasSelection = 1 <= selectedElements.length;
 
 	return (
 		<Background>
 			{ currentPage && currentPage.elements.map( ( { id, ...rest } ) => {
-				const isSelected = Boolean( selectedElement && selectedElement.id === id );
-
+				const isSelected = Boolean( selectedElements.filter( ( { id: selectedId } ) => id === selectedId ).length );
```

Comment:
I know there are some changes coming in that area, but use can use `selectedElementIds` property to simply test `selectedElementIds.includes(id)`.